test/integration/handlers/transfers/handlers.test.js

```diff
@@ -319,6 +319,151 @@ const prepareTestData = async (dataObj) => {
+    })
+    await ParticipantLimitHelper.prepareLimitAndInitialPosition(fxp.participant.name, {
+      currency: dataObj.targetAmount.currency,
+      limit: { value: dataObj.fxp.limit }
+    })
+    await ParticipantFundsInOutHelper.recordFundsIn(payer.participant.name, payer.participantCurrencyId2, {
+      currency: dataObj.sourceAmount.currency,
+      amount: 10000
+    })
+
+    for (const name of [payer.participant.name, fxp.participant.name]) {
+      await ParticipantEndpointHelper.prepareData(name, 'FSPIOP_CALLBACK_URL_TRANSFER_POST', `${dataObj.endpoint.base}/transfers`)
+      await ParticipantEndpointHelper.prepareData(name, 'FSPIOP_CALLBACK_URL_TRANSFER_PUT', `${dataObj.endpoint.base}/transfers/{{transferId}}`)
+      await ParticipantEndpointHelper.prepareData(name, 'FSPIOP_CALLBACK_URL_TRANSFER_ERROR', `${dataObj.endpoint.base}/transfers/{{transferId}}/error`)
+      await ParticipantEndpointHelper.prepareData(name, 'FSPIOP_CALLBACK_URL_BULK_TRANSFER_POST', `${dataObj.endpoint.base}/bulkTransfers`)
+      await ParticipantEndpointHelper.prepareData(name, 'FSPIOP_CALLBACK_URL_BULK_TRANSFER_PUT', `${dataObj.endpoint.base}/bulkTransfers/{{id}}`)
+      await ParticipantEndpointHelper.prepareData(name, 'FSPIOP_CALLBACK_URL_BULK_TRANSFER_ERROR', `${dataObj.endpoint.base}/bulkTransfers/{{id}}/error`)
+      await ParticipantEndpointHelper.prepareData(name, 'FSPIOP_CALLBACK_URL_QUOTES', `${dataObj.endpoint.base}`)
+      await ParticipantEndpointHelper.prepareData(name, Enum.EndPoints.FspEndpointTypes.FSPIOP_CALLBACK_URL_FX_QUOTES, `${dataObj.endpoint.base}`)
+      await ParticipantEndpointHelper.prepareData(name, Enum.EndPoints.FspEndpointTypes.FSPIOP_CALLBACK_URL_FX_TRANSFER_POST, `${dataObj.endpoint.base}/fxTransfers`)
+      await ParticipantEndpointHelper.prepareData(name, Enum.EndPoints.FspEndpointTypes.FSPIOP_CALLBACK_URL_FX_TRANSFER_PUT, `${dataObj.endpoint.base}/fxTransfers/{{commitRequestId}}`)
+      await ParticipantEndpointHelper.prepareData(name, Enum.EndPoints.FspEndpointTypes.FSPIOP_CALLBACK_URL_FX_TRANSFER_ERROR, `${dataObj.endpoint.base}/fxTransfers/{{commitRequestId}}/error`)
+    }
+
+    const transferPayload = {
+      commitRequestId: randomUUID(),
+      determiningTransferId: randomUUID(),
+      condition: 'YlK5TZyhflbXaDRPtR5zhCu8FrbgvrQwwmzuH0iQ0AI',
+      expiration: new Date((new Date()).getTime() + (24 * 60 * 60 * 1000)), // tomorrow
+      initiatingFsp: payer.participant.name,
+      counterPartyFsp: fxp.participant.name,
+      sourceAmount: {
+        currency: dataObj.sourceAmount.currency,
+        amount: dataObj.sourceAmount.amount
+      },
+      targetAmount: {
+        currency: dataObj.targetAmount.currency,
+        amount: dataObj.targetAmount.amount
+      }
+    }
+
+    const fxPrepareHeaders = {
+      'fspiop-source': payer.participant.name,
+      'fspiop-destination': fxp.participant.name,
+      'content-type': 'application/vnd.interoperability.transfers+json;version=1.1'
```

Comment:
maybe, the content-type should be `application/vnd.interoperability.**fx**Transfers+json;version=1.1` ?

Reply:
done